test/test_generated_model_sim_environments.py

```diff
@@ -1,11 +1,8 @@
 import os
```

Comment:
I thought you weren't going to rename tests any more???

Reply:
I needed a change to rebuild :D
Have given all 3 their original names now